app/serializers/event_stream_serializers/classification_serializer.rb

```diff
@@ -7,12 +7,30 @@ def self.serialize(classification, options = {})
       Serialization::V1Adapter.new(serializer, options)
     end
 
-    attributes :id, :created_at, :updated_at, :user_ip, :annotations, :metadata
+    attributes :id, :created_at, :updated_at, :user_ip, :workflow_version, :gold_standard,
+               :expert_classifier, :annotations, :metadata
 
-    belongs_to :project,  serializer: EventStreamSerializers::ProjectSerializer
-    belongs_to :user,     serializer: EventStreamSerializers::UserSerializer
-    belongs_to :workflow, serializer: EventStreamSerializers::WorkflowSerializer
-    has_many   :subjects, serializer: EventStreamSerializers::SubjectSerializer
+    belongs_to :project,          serializer: EventStreamSerializers::ProjectSerializer
+    belongs_to :user,             serializer: EventStreamSerializers::UserSerializer
+    belongs_to :workflow,         serializer: EventStreamSerializers::WorkflowSerializer
+    belongs_to :workflow_content, serializer: EventStreamSerializers::WorkflowContentSerializer
+    has_many   :subjects,         serializer: EventStreamSerializers::SubjectSerializer
+
+    def workflow
+      version = workflow_version.split(".")[0].to_i
```

Comment:
Couldn't really come up with a good abstraction. I'd want to call this `workflow_version` but that's what the combined workflow+workflow_content version is already called. :(